create/how_to/updating.md

```diff
@@ -0,0 +1,146 @@
+
+This process can take some time. Once the status is `"done"`, move on to the next step.
+
+### Step 3: Import the dump
+
+Now that you’ve got your dump, you just need to [install the latest version of MeiliSearch](/learn/getting_started/installation.md#download-and-launch) and import the dump at launch, like so:
+
+```bash
+# Install MeiliSearch in the local directory
+curl -L https://install.meilisearch.com | sh
+
+# Launch MeiliSearch and import the specified dump file
+./meilisearch --import-dump /dumps/name_of_your_dump.dump
+```
+
+> Importing a dump is the same process as indexing documents. This can take some time and cause a spike in RAM usage. To save RAM, use a [smaller batch size](/reference/features/configuration.md#dump-batch-size).
+
+Finally, don’t forget to set `displayedAttributes` back to its previous value, if necessary.
+
+Congratulations! You have successfully migrated your MeiliSearch database to the latest version! 🎉
+
+## Updating from v0.14.0 or below
+
+Since these versions predate the [dumps feature](/reference/features/dumps.md), the best solution is to export your documents and your [index settings](/reference/features/settings.md) as JSON files.
+
+If you don’t need to preserve index settings, skip directly to [step 2](#step-2-set-all-fields-as-displayed-attributes).
+
+### Step 1: Save your settings
+
+First, use the [get settings](/reference/api/settings.md#get-settings) endpoint to retrieve the [settings](/reference/features/settings.md) of any indexes you want to preserve, and save them to a file using the method you prefer.
+
+<RouteHighlighter method="GET" route="/indexes/:index_uid/settings"/>
+
+For simplicity's sake, this example uses cURL.
+
+```bash
+# the -o option saves the output as a local file
+curl -X GET “http://127.0.0.1:7700/indexes/:index_uid/settings” -o mysettings.json
+```
+
+It is important to first export your settings before exporting documents, since retrieving the documents in their entirety may require modifying the settings.
+
+### Step 2: Set all fields as displayed attributes
+
+In order to retrieve all the fields present in your documents, all fields must be set as `displayedAttributes`.
+
+By default, all fields are included in `displayedAttributes`. Still, it's a good idea to verify your settings before continuing. Do so using the [get displayed attributes](/reference/api/displayed_attributes.md#get-displayed-attributes) endpoint:
+
+<RouteHighlighter method="GET" route="/indexes/:index_uid/settings/displayed-attributes"/>
+
+If the returned value is `["*"]`, you can move on to the next step.
+
+If not, use the [reset displayed-attributes](/reference/api/displayed_attributes.md#reset-displayed-attributes) endpoint.
+
+<RouteHighlighter method="DELETE" route="/indexes/:index_uid/settings/displayed-attributes"/>
+
+Now that all fields are **displayed**, you can proceed to save your documents without risking any information loss.
+
+### Step 3: Save your documents
```

Comment:
We need to add a warning or a note in this section to remind users to raise the limit. If they don't raise it they will only have 20 documents in their file!

Reply:
Great catch, thanks!